gems/sorbet-runtime/lib/types/helpers.rb

```diff
@@ -24,6 +24,23 @@ def sealed!
     Private::Sealed.declare(self, Kernel.caller(1..1)&.first&.split(':')&.first)
   end
 
+  # Specify an inclusion or inheritance requirement for `self`.
+  #
+  # Example:
+  #
+  #   module MyHelper
+  #     extend T::Helpers
+  #
+  #     requires_ancestor Kernel
+  #   end
+  #
+  #   class MyClass < BasicObject # error: `MyClass` must include `Kernel` (required by `MyHelper`)
+  #     include MyHelper
+  #   end
+  #
+  # TODO: implement the checks in sorbet-runtime.
```

Comment:
What's your plan for sorbet-runtime?

Reply:
No, it does not require anything beside this definition. The requirement will just not be checked at runtime.

I think @paracycle was interested in providing the runtime behaviour in a follow-up PR.